journal-letters/modernized/03-chapter-2.adoc

```diff
@@ -88,8 +103,15 @@ year of my life it would be truly profitable.
 The meetings for discipline, as well as those for worship, were very consolatory;
 my spirit was much broken and contrited before the Most High, and I walked in great fear.
 
+[.small-break]
+'''
```

Comment:
I wonder if maybe instead of a small-break here, making the next sentence an `.offset` would be better. 🤔